test/common/ssl/connection_impl_test.cc

```diff
@@ -12,18 +12,27 @@
 #include "test/mocks/runtime/mocks.h"
 #include "test/mocks/stats/mocks.h"
 #include "test/mocks/server/mocks.h"
+#include "test/test_common/environment.h"
 
 using testing::_;
 using testing::Invoke;
 
 namespace Ssl {
 
-static void testUtil(std::string client_ctx_json, std::string server_ctx_json,
-                     std::string expected_digest, std::string expected_uri) {
+namespace {
+
+Json::ObjectPtr jsonLoad(const std::string& raw_json) {
```

Comment:
Should we put all the transforms like this in one place? Per our offline chat, if we eventually switch to using jinja, it will be easier to figure out what is going on?